pkg/plugins/label/label.go

```diff
@@ -40,7 +40,7 @@ var (
 		ConfigHelpProvider: configHelp,
 		Commands: []plugins.Command{{
 			Prefix: "remove-",
-			Name:   "area|committee|kind|language|priority|sig|team|triage|wg|label",
+			Name:   "area|committee|kind|language|priority|sig|team|triage|wg|label|risk",
```

Comment:
Do you think risk should be a constrained label rather than a wildcard label?

Risk having a completely open ended label would limit extra features being built off of this in future

Reply:
It would make it so it would have to be `/label risk/low` which is a little more inconvenient then `/risk low` and the other issue is in gitlab `/label` is reserved by gitlab itself so `/label` never makes it here

Reply:
is see `risk` similar to `priority` which is already in the list.